### PR TITLE
Auto-update and reboot as needed using dnf-automatic

### PR DESCRIPTION
**Pull request by hartwork**
@ulm two known todos:
- Rebase onto #7 after it's merged
- Duplicate to kiwi3 after some review

What do you think?

### CODE REVIEW

**Comment by hartwork** [comment]
@ulm PS: by default the timer will hit every day at 6am:

```console
# systemctl cat dnf-automatic.timer | grep OnCalendar
OnCalendar=*-*-* 6:00
```

I can make it a different time if preferred.

File: ansible/playbook-setup-quince.yml [hunk 1]
```diff
@@ -26,7 +26,11 @@
     package:
       state: present
       name:
+    # Maintenance
+      - dnf-automatic
+      - python3-tracer
     # Debugging tools
+      - bash-completion
```
Comment (ulm) [inline ansible/playbook-setup-quince.yml:33]: Why is `bash-completion` needed for this?

**Review by ulm — COMMENTED** [review]
Go for it, and don't wait for #7. I can rebase it as necessary.

File: ansible/playbook-setup-quince.yml (hunk 1 above)
Comment (hartwork) [reply ansible/playbook-setup-quince.yml:33]: I noticed absence of `bash-completion` while paying with systemd. I can drop this or make a dedicated commit — what do you prefer?

Comment (ulm) [reply ansible/playbook-setup-quince.yml:33]: Yes, please put it in a separate commit if it's unrelated.

Another question, in #7 I put the common file in a new directory `files/common/` while your files are directly in `files/`. Presumably we should unify this? (I don't have any strong preference for either.)

Comment (hartwork) [reply ansible/playbook-setup-quince.yml:33]: @ulm good point. With the current theme, it takes (minimal but) effort to classify "common" as a "magic" host, as a non-host. After consideration, I'd prefer without "common".

Comment (hartwork) [reply ansible/playbook-setup-quince.yml:37]: > Yes, please put it in a separate commit if it's unrelated.

@ulm done, pushed

File: ansible/files/restart-as-needed.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
```
Comment (ulm) [inline ansible/files/restart-as-needed.sh:1]: I can't say that I'm a friend of the `/usr/bin/env` hack, but well...

(It's not as portable as it claims to be: POSIX doesn't allow any arguments for the command.)

**Review by ulm — APPROVED** [review]

File: ansible/files/restart-as-needed.sh (hunk 2 above)
Comment (hartwork) [reply ansible/files/restart-as-needed.sh:1]: @ulm it's not as flexible as one could wish, but let's take that discussion offline…

Comment (hartwork) [reply ansible/files/restart-as-needed.sh:1]: @ulm thanks for the review, merging…

**Pull request merged**